### PR TITLE
Update JSON Schema Faker

### LINKED ISSUE #715: Prism server always returns empty body response when "addtionalProperties" is false
**Describe the bug**
When additionalProperties is false, Prism server always returns empty body response.

**To Reproduce**

1. OpenAPI specification:

```
openapi: 3.0.0
info:
  title: Sample API
  version: v1
paths:
  '/services/resources/{resource_id}':
    get:
      summary: Get a resource info.
      description: returns a single resource.
      operationId: getResource
      responses:
        '200':
          description: A resource
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Resource'
        '404':
          description: Not found.
components:
  schemas:
    Resource:
      type: object
      properties:
        name: 
          type: string
        tags:
          type: string
        createTime:
          type: string
          format: date-time
      additionalProperties: false
```

2. Start prism mock server and run curl http://localhost:4010/services/resources/5

**Current behavior**
 {}

**Expected behavior**
{
    "name": "aliquip",
    "tags": "",
    "createTime": "1985-06-04T11:15:57.166Z"
}

**Environment:**

Library version: 3.1.1
OS: MacOS Darwin Kernel Version 18.7.0

**Additional context**
The additionalProperties false means the properties not in the properties listed are not allowed in the request.  (Block the Invalid request). But the mock server should behave the same as addtionalProperties is true.

### PR DESCRIPTION
**Pull request by XVincentX**
Closes #715

### CODE REVIEW

File: test-harness/specs/dynamic_json_responses_additionalProperties.oas3.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+components:
+  schemas:
+    Resource:
+      type: object
+      properties:
+        name:
+          type: string
+        tags:
+          type: string
+      additionalProperties: false
```
Comment (lag-of-death) [inline test-harness/specs/dynamic_json_responses_additionalProperties.oas3.txt]: @XVincentX, would it be feasible to add:
1. a tests with `additionalProperties: true`
2. a test with `additionalProperties` as an obj
?

Comment (XVincentX) [reply test-harness/specs/dynamic_json_responses_additionalProperties.oas3.txt]: Done. It didn't really change the final output though.

**Comment by lag-of-death** [comment]
@XVincentX, what about the case of `additionalProperties: true`?

**Comment by XVincentX** [comment]
Rename to what?

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Killing sub-process only if Prism is running in multi-process mode [#645](https://github.com/stoplightio/prism/pull/645)
 - UUIDs are never generated as URNs [#661](https://github.com/stoplightio/prism/pull/661)
 - Core types are now correctly referenced in the HTTP package, restoring the type checks when using the package separately [#701](https://github.com/stoplightio/prism/pull/701)
+- By upgrading Json Schema Faker to the latest version, now the schemas with `additionalProperties:false` will be correctly handled when dynamic mocking is enabled [#719](https://github.com/stoplightio/prism/pull/719)
```
Comment (lag-of-death) [inline CHANGELOG.md]:
```suggestion
- By upgrading Json Schema Faker to the latest version, now the schemas with `additionalProperties:false` / `additionalProperties:true` / `additionalProperties:object` will be correctly handled when dynamic mocking is enabled [#719](https://github.com/stoplightio/prism/pull/719)
```

**Comment by lag-of-death** [comment]
@XVincentX, to https://github.com/stoplightio/prism/pull/719/files#r336916791

**Pull request merged**